assets/i18n/id.json

```
@@ -0,0 +1,262 @@
+{
+    "@metadata": {
+        "authors": [
+            "Kim Shibetoshi",
+            "getlocalization contributors",
+            "etc"
+        ],
+        "last-updated": "2021-04-14",
+        "locale": "id",
+        "message-documentation": "Doge-wow-me! DG8j7EoMG8BgWnijXKZ12oZHs7KHmtGmYd Such thanks!",
```

To be honest, I'm against that address in there.

---

```suggestion
        "message-documentation": "qqq",
```
Substituting to match English